mac.sh

```diff
@@ -138,7 +138,7 @@ install_or_update_homebrew() {
 
 install_latest_ruby() {
   append_to_zshrc 'eval "$(rbenv init - zsh --no-rehash)"' 1
-  ruby_version="2.3.4"
+  ruby_version="2.3.7"
```

Comment:
"latest" 😂 

is there a reason it's not 2.3.8?

Reply:
I was wondering about that! The base image for the monolith dockerfile is set to `pg-ruby:2.3.7-rails4`, so didn't want to go past that.